src/init.cpp

```diff
@@ -410,7 +410,7 @@ void SetupServerArgs(NodeContext &node) {
         "-min", "-resetguisettings", "-rootcertificates=<file>", "-splash",
         "-uiplatform",
         // TODO remove after the November 2020 upgrade
-        "-axionactivationtime"};
+        "-axionactivationtime", "-exodusactivationtime"};
```

Comment:
Should probably rename this so we ensure we get merge conflicts on any activation changes of theirs. That's what I did with the actual variable.

Reply:
I guess what I'm saying is to remove `"-axionactivationtime"` and add this as a separate PR.